src/Tickets/Commerce/Gateways/PayPal/REST/Order_Endpoint.php

```diff
@@ -177,7 +177,8 @@ public function handle_update_order( WP_REST_Request $request ) {
 			return new WP_Error( 'tec-tc-gateway-paypal-nonexistent-order-id', null, $order );
 		}
 
-		$paypal_capture_response = tribe( Client::class )->capture_order( $paypal_order_id );
+		$payer_id = $request->get_param( 'payerID' ) ? $request->get_param( 'payerID' ) : '';
```

Comment:
You can simplify this. It would result in `null` if the param doesn't exist.
```suggestion
		$payer_id = $request->get_param( 'payerID' );
```